src/test/resources/json/get_consignment_for_export.json

```diff
@@ -8,7 +8,6 @@
       "createdDatetime": "2021-01-15T07:47:19Z",
       "transferInitiatedDatetime": "2021-01-15T07:47:19Z",
       "userid": "b2657adf-6e93-424f-b0f1-aadd26762a96",
-      "consignmentReference": "consignmentReference-1234",
```

Comment:
Was this removed accidentally, or is there a reason the API response wouldn't include this?

Reply:
Thanks for spotting. Accidentally removed this.